uast/uast.go

```diff
@@ -30,6 +30,41 @@ const (
 	Expression
 	Statement
 
+	// Comparison operators. Usually inside one of the Condition nodes, but could also be in an
+	// an expression of boolean value by itself. The tested expressions will be its siblings.
+
+	// OpCompEquality is the operator that tests for logical equality between two expressions
+	OpCompEquality
```

Comment:
Yeah, Boolean and Bitwise need their qualifiers to avoid conflicts but comparison operators doesn't really need it. Will update the PR.